metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -140,6 +140,7 @@ def kill(ctx, run_id, user, my_runs):
 @click.option("--shared-memory", help="Shared Memory requirement for AWS Batch.")
 @click.option("--max-swap", help="Max Swap requirement for AWS Batch.")
 @click.option("--swappiness", help="Swappiness requirement for AWS Batch.")
+@click.option("--inferentia", help="Inferentia requirement for AWS Batch.")
```

Comment:
Is my understanding correct that the Batch Compute Environment needs to be entirely composed of Inf1 instances to have any guarantee that the job will launch on top of the inf1 instance?

Reply:
Yes, that is my understanding also